server/Mistakes.Journal.Api/Api/Mistakes/Controllers/MistakesController.cs

```diff
@@ -70,39 +135,33 @@ public async Task<ActionResult<MistakeWebModel>> CloneMistake(Guid mistakeId)
         public async Task<ActionResult<MistakeWebModel>> GetMistakes([FromQuery] PagingParameters pagingParameters)
         {
             var mistakes = await _dataContext.Set<Mistake>()
-                .Include(m => m.Tips)
                 .Skip(pagingParameters.StartAt)
                 .Take(pagingParameters.MaxResults)
+                .Include(m => m.Tips)
+                .Include(m => m.MistakeLabels)
+                .ThenInclude(m => m.Label)
```

Comment:
Could you nest `ThenIncludes`?
I mean
```suggestion
                .Include(m => m.MistakeLabels)
                    .ThenInclude(m => m.Label)
```